homeassistant/components/comelit/alarm_control_panel.py

```diff
@@ -92,7 +92,9 @@ def __init__(
     @property
     def _area(self) -> ComelitVedoAreaObject:
         """Return area object."""
-        return self.coordinator.data[ALARM_AREAS][self._area_index]
+        return cast(
```

Comment:
If we need to cast everything to the correct type it suggests that the underlying data lacks type annotations. We don't really want to apply strict typing for those integrations. The underlying data should preferably be typed first.